supabase/migrations/37_alerts.sql

```diff
@@ -0,0 +1,136 @@
+create table alert_subscriptions (
+  like internal._model including all,
+
+  catalog_prefix    catalog_prefix                 not null,
+  email             text
+);
+alter table alert_subscriptions enable row level security;
+
+create policy "Users access subscriptions for the prefixes they admin"
+  on alert_subscriptions as permissive
+  using (exists(
+    select 1 from auth_roles('admin') r where catalog_prefix ^@ r.role_prefix
+  ));
+
+grant select, insert, update, delete on alert_subscriptions to authenticated;
+
+create table alert_data_processing (
+  catalog_name           catalog_name not null,
+  evaluation_interval    interval     not null,
+  primary key (catalog_name)
+);
+alter table alert_data_processing enable row level security;
+
+create policy "Users access alerts for admin-authorized tasks"
+  on alert_data_processing as permissive
+  using (exists(
+    select 1 from auth_roles('admin') r where catalog_name ^@ r.role_prefix
+  ));
+
+grant update (evaluation_interval) on alert_data_processing to authenticated;
+grant select, insert, delete on alert_data_processing to authenticated;
+
+create table alert_history (
+  alert_type      text         not null,
+  catalog_name    catalog_name not null,
+  fired_at        timestamptz  not null,
+  resolved_at     timestamptz,
+  arguments       json         not null,
+  primary key (alert_type, catalog_name)
+);
+alter table alert_history enable row level security;
+
+create policy "Users access alert history for admin-authorized tasks"
+  on alert_history as permissive
+  using (exists(
+    select 1 from auth_roles('admin') r where catalog_name ^@ r.role_prefix
+  ));
+
+grant select on alert_history to authenticated;
+
+create view internal.alert_data_processing_firing as
```

Comment:
I think this view could use some test cases. See [this example](https://github.com/estuary/flow/blob/master/supabase/tests/live_specs_ext.test.sql) for how to write tests using pgTap, and LMK if you have any questions.